genologics/descriptors.py

```diff
@@ -11,7 +11,7 @@
 try:
```

Comment:
We can get rid of the try here. It's doing the same thing as in the except handler.

Reply:
... my compiler tells me these aren't even used yet.

Reply:
Ok